src/components/MatchLoading/MatchLoading.jsx

```diff
@@ -0,0 +1,29 @@
+import Loading from '../LoadingPage/Loading';
+import * as S from './style';
+
+const MatchLoading = () => {
+    return (
+        <div>
+            <Loading></Loading>
```

Comment:
```suggestion
            <Loading />
```

이런식으로 축약 가능합니다. 참고해주세요.

Reply:
수정 했습니다